src/main/java/com/javadiscord/jdi/internal/models/guild/AutoModerationRuleObject.java

```diff
@@ -0,0 +1,21 @@
+package com.javadiscord.jdi.internal.models.guild;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+// TODO: relook at how AutoModeration handler's are made and possibly redo them (alot was missing)
```

Comment:
did u completed this todo or is this for future dev?

Reply:
I did do that, just forgot to remove